docs/tools/aleph-client/documentation.md

```diff
@@ -0,0 +1,942 @@
+# Aleph CLI
+
+**Usage**:
+
+```console
+$ aleph aleph [OPTIONS] COMMAND [ARGS]...
+```
+
+**Options**:
+
+* `--install-completion`: Install completion for the current shell.
+* `--show-completion`: Show completion for the current shell, to copy it or customize the installation.
+* `--help`: Show this message and exit.
+
+**Commands**:
+
+* `about`: Display the informations of Aleph CLI
```

Comment:
You can remove the "s" in information, the word is always singular, and if you don't need to specify specific information you can also remove "the" in front of the word.